src/organizations/organizations.ts

```diff
@@ -0,0 +1,13 @@
+import { IResponse, IStringToStringMap } from '../lib/router';
+
+import { IContext } from '../app/context';
+
+import organizationsTemplate from './organizations.njk';
+
+export async function listOrganizations(ctx: IContext, _params: IStringToStringMap): Promise<IResponse> {
```

Comment:
I think `IStringToStringMap` would be better named something like `IParams` ... I don't think there's much value in reusing these types of types